task_sdk/src/airflow/sdk/execution_time/comms.py

```diff
@@ -104,6 +105,30 @@ def from_asset_response(cls, asset_response: AssetResponse) -> AssetResult:
         return cls(**asset_response.model_dump(exclude_defaults=True), type="AssetResult")
 
 
+class AssetEventCollectionResult(AssetEventCollectionResponse):
```

Comment:
Just `AssetEventResult`?

Reply:
how about AssetEvent`s`Result?

Reply:
I updated it as `AssetEventsResult`. I'll mark it as resolved. but we can discuss more if this name is not desired